portingdb/templates/index.html

```diff
@@ -55,7 +55,11 @@ <h2 id="in-progress">Porting in Progress</h2>
             {% if mispackaged_packages %}
                 <h2 id="mispackaged">Mispackaged packages</h2>
                 <div>
-                    {{ len(mispackaged_packages) }} packages were found to have some kind of packaging problem:
+                    {{ len(mispackaged_packages) }} packages were found to have some kind of packaging problem.
+                </div>
+                <div>
+                    To find the best package to port, see <a href="{{ url_for('mispackaged') }}">packages ordered by last BugZilla comment</a>.<br/>
```

Comment:
Huh, now you stumped me. I think it has to do with my love for CamelCase that somehow made me believe it was the official spelling of the name. :-o

It shall be rectified!